### DISCUSSION TITLE
I want to attend MONAI MICCAI Bootcamp

### DISCUSSION THREAD

**Discussion by tashrifbillah** [Project-MONAI/MONAI#2989] [General]
I came to know about this event after the registration deadline has passed:
https://www.gpuhackathons.org/event/monai-miccai-bootcamp-2021

It is scheduled to run September 22-24, 2021. I have sent an email to `monai.contact@gmail.com` expressing my interest to attend but did not hear back. So I am not sure if that inbox is monitored.

If you are one of the organizers and can still let people attend this `virtual`<sup>*</sup> event, please let me know.

**Comment by wyli** [comment]
cc @zephyrie @pdogra89

**Reply by ramchandracheke** [reply]
I am also interested! Please let me know, if I can still attend it!
